spec/time_of_day_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+describe Timely::TimeOfDay do
+  subject(:time_of_day) { Timely::TimeOfDay.new(4, 52) }
+
+  its(:hour) { should == 4 }
+  its(:minute) { should == 52 }
+
+  context 'asking for that time on a date' do
+    subject(:time) { time_of_day.on_date(Date.new(2014, 1, 1)) }
+    its(:month) { should == 1 }
+    its(:hour) { should == 4 }
+    its(:min) { should == 52 }
+  end
+
+  context 'can initialize from a time' do
+    it { should == Timely::TimeOfDay.from_time(DateTime.new(2014, 2, 3, 4, 52)) }
```

Comment:
Line is too long. [81/80]<br>Do not use `DateTime.new` without zone. Use one of `Time.zone.local`, `DateTime.new.current`, `DateTime.new.in_time_zone`, `DateTime.new.utc`, `DateTime.new.getlocal`, `DateTime.new.iso8601`, `DateTime.new.jisx0301`, `DateTime.new.rfc3339`, `DateTime.new.to_i`, `DateTime.new.to_f` instead.